pixi.toml

```diff
@@ -90,6 +90,9 @@ nbval = "*"
 [feature.doc]
 channels = ["pyviz"]
 
+[feature.doc.activation.env]
+PARAM_SIGNATURE = "1"
```

Comment:
We would need this to be enabled in our other packages, maybe just upstream it directly to nbsite.

Reply:
So they can also take advantage of the updated signatures in their documentation. 

Could also be that I'm confused.

Reply:
I can only think of Panel for a library that has custom Parameters, like `Margin`. It doesn't look like it's documented in Panel's API ref. Not worth the trouble of changing anything in these packages/nbsite until the need comes up (the code to dynamically update the signature has only been tested on Param's parameters, I don't know if it works for other Parameters).